saleor/plugins/webhook/tasks.py

```diff
@@ -81,7 +78,7 @@ def create_deliveries_for_subscriptions(
     :param requestor: used in subscription webhooks to generate meta data for payload.
     :return: List of event deliveries to send via webhook tasks.
     """
-    if event_type not in SUBSCRIBABLE_EVENTS:
+    if event_type not in list(WEBHOOK_TYPES_MAP.keys()):
```

Comment:
You can simplify this:
```suggestion
    if event_type not in WEBHOOK_TYPES_MAP:
```

Reply:
Right 🤦